mobile/build.gradle

```diff
@@ -122,6 +123,17 @@ def readProguardFiles() {
     return proguardFiles
 }
 
+tasks.withType(Checkstyle) {
+    configFile = rootProject.file('config/checkstyle/checkstyle.xml')
```

Comment:
@garg-anuj Gradle guidelines say that checkstyle.xml should be in a folder called config. I can transfer the file if you want.